model/riscv_vmem.sail

```diff
@@ -164,7 +164,30 @@ function pt_walk(sv_params,
                 }
               }
               else {
+                let mask_bits = level * sv_params.pte_PPN_j_size_bits;
```

Comment:
I haven't gone through the logic properly but are you sure this is right? `level` is guaranteed to be 0 here.